File: tourneyAPI/Models/Entities/Characters/Byleth.cs
namespace Entities;

using Enums;

// Defines the competitive profile metadata for this playable character.
public class Byleth : Character
{
    // Initializes this character's default competitive attributes for matchmaking and tier logic.
    public Byleth ()
    {
        Id = CharacterId.Byleth;
        characterName = CharacterName.BYLETH;
        archetype = Archetype.PRECISION;
        fallSpeed = FallSpeed.FAST_FALLERS;
        weightClass = WeightClass.LIGHTWEIGHT;
        tierPlacement = TierPlacement.B;
    }

}

File: tourneyClient/src/models/entities/Characters/Byleth.ts
import { CharacterId } from "@/models/Enums/CharacterId";
import { Character } from "../Character"
import { CharacterName } from "@/models/Enums/CharacterName";
import { Archetype } from "@/models/Enums/Archetype";
import { FallSpeed } from "@/models/Enums/FallSpeed";
import { WeightClass } from "@/models/Enums/WeightClass";
import { TierPlacement } from "@/models/Enums/TierPlacement";

export const Byleth: Character =
{
    id: CharacterId.Byleth,
    characterName: CharacterName.BYLETH,
    archetype: Archetype.PRECISION,
    fallSpeed: FallSpeed.FAST_FALLERS,
    weightClass: WeightClass.LIGHTWEIGHT,
    tierPlacement: TierPlacement.B
}

export default Byleth; 